scripts/get_models.py

```diff
@@ -0,0 +1,32 @@
+import os
+import json
+import time
+import requests
+
+def main():
+    with open('../data/api/brands.json') as f:
+        data = f.read()
+
+    brands = json.loads(data)['brands']
+
+    API_KEY = os.environ['API_KEY']
+
+    for brand in brands:
+        brand_name = brand['name']
+        brand_id = brand['id']
+        print(brand_name)
+
+        r = requests.post('http://api.watchbase.com/v1/watches', data = {'key': API_KEY,
```

Comment:
would be nice not to have that in repo ;)

Reply:
Makes sense